### ISSUE TITLE
[BUG] Weapons & Clothing - Price cannot go higher than 1mill

### ISSUE THREAD

**Issue by DarkSilencerZA** [AsYetUntitled/Framework#33]
Say you add an RPG to the shop and want to make it 15mill - it jumbles the pricing - it's still purchasable, however it can be abused. If you have no cash on hand, and purchase it - it gives you cash instead.

**Comment by tkcjesse** [comment]
Are you sure... Aren't you using my debit card system...?

**Comment by DarkSilencerZA** [comment]
Yep I am using the Debit Cards - but it doesn't make a difference. The shops don't support higher than 1mill or so, else it bugs out.

Not sure how relevant this is, but:

https://github.com/TAWTonic/Altis-Life/issues/3

**Comment by BoGuu** [comment]
https://github.com/AsYetUntitled/Framework/blob/master/Altis_Life.Altis/core/shops/fn_weaponShopSelection.sqf#L23

Posted for reference

**Comment by Jawshy** [comment]
SITREP: 
1. Is this issue reproducible in the current build?

**Comment by SimZor** [comment]
@DarkSilencerZA ^

**Comment by trinitynet** [comment]
![screenshot_6](https://cloud.githubusercontent.com/assets/8790525/19829337/91cdd542-9dd5-11e6-8379-b504d7e17ea0.png)
i can confirm this is an issue atm @Jawshy , it also gives me the money that i defined and not taken it.
